addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -1164,8 +1160,8 @@ registerModel({
             if (this.messaging.userSetting.rtcConfigurationMenu.isRegisteringKey) {
                 return;
             }
-            if (this._pushToTalkTimeoutId) {
-                browser.clearTimeout(this._pushToTalkTimeoutId);
+            if (this.pushToTalkTimeout) {
```

Comment:
useless condition, `clearTimeout` accepts invalid ids (and does nothing in that case) according to documentation